### ISSUE TITLE
Liquibase diffChangeLog generation is broken in 1.7.1

### ISSUE THREAD

**Issue by AlexKovynev** [quarkusio/quarkus#11692]
**Describe the bug**
I use hibernate-liquibase as a plugin for maven or as a plugin in command-line. In both variants fail in 1.7 but work in 1.6.1.

Unexpected error running Liquibase: Duplicate event listener found
liquibase.exception.DatabaseException: org.hibernate.event.service.spi.EventListenerRegistrationException: Duplicate event listener found.

It's like some events now is auto registered.

**Expected behavior**
Generate something.

**Actual behavior**
Exception.

**To Reproduce**
1. Liquibase diffChangeLog

**Environment (please complete the following information):**
 - Output of `java -version`: openjdk version "11.0.7" 2020-04-14
 - Quarkus version or git rev: 1.7.1
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 3.6.3

**Comment by geoand** [comment]
Can you please provide a minimal reproducer?

Thanks

**Comment by gsmet** [comment]
I think the issue is similar to https://github.com/quarkusio/quarkus/pull/11679 (on Zulip, the OP had the same exception IIRC)

**Comment by gsmet** [comment]
And we have a reproducer there but another one might be helpful.

**Comment by AlexKovynev** [comment]
It is a problem with a quarkus-hibernate-validator. If i downgrade version to 1.6.1.Final then command work.

Small reproducer with one model and h2 in-memory datasource.

 ./mvnw clean install liquibase:diff

[reproducer.zip](https://github.com/quarkusio/quarkus/files/5140709/reproducer.zip)

**Comment by kunieda-o** [comment]
I also found the error like this. 1.7.0 works well but 1.7.1 doesn't work well.
```
Map<String, Object> properties = new HashMap<>();
properties.put(AvailableSettings.DRIVER, "org.h2.Driver");
properties.put(AvailableSettings.URL, "jdbc:h2:mem:test");
PersistenceUnitInfoAdapter unitInfo = new PersistenceUnitInfoAdapter("test");
EntityManagerFactory emf = new HibernatePersistenceProvider().createContainerEntityManagerFactory(unitInfo, properties);
```
I use above in Unit test.

**Comment by geoand** [comment]
Thanks for reporting. I'll have a look soon

**Comment by geoand** [comment]
It turns out that different behavior between `1.7.0.Final` and `1.7.1.Final` is caused by commit e68c5f64c86aedaebf6b59c21b25f225de087181 of #/11494.

Basically what happens is that because that commit added the `org.hibernate.integrator.spi.Integrator` file to the HV extension, now HV seems to be loaded twice when Hibernate is booted manually (which is the case for the Liquibase plugin).

This use case is not something we ever supported and the fact that it used to work is pure coincidence IMHO. 

Since I am no Hibernate expert, I'll hand this one over to @gsmet and @Sanne

**Comment by gsmet** [comment]
It's going to be fixed in 1.7.2 but keep in mind that we don't support creating EMF manually. You lose most of Hibernate ORM Quarkus optimizations by creating the EMF manually. And there's a good chance it won't work at all or work differently (except if you carefully apply all the settings applied by Quarkus).

**Comment by AlexKovynev** [comment]
Ok, thank you, unfortunately liquibase-hibernate does not support quarkus, only spring and pu from dedicated xml. And as i know we have not any analogs in the market now for generate migrations automaticaly.
